server/game/cards/02_SHD/units/KihraxzHeavyFighter.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class KihraxzHeavyFighter extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '4721657243',
+            internalName: 'kihraxz-heavy-fighter'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addOnAttackAbility({
+            title: 'You may exhaust another friendly unit. If you do, this unit gets +3/+0 for this attack.',
+            targetResolver: {
+                optional: true,
```

Comment:
We try to put the `optional` as high as possible in the property hierarchy, in this case it can go directly on the ability (same level as `title` and `targetResolver`)

Reply:
Does this really make sense in this case? If I move it up, it won't auto skip if there are no units to tap